okhttp/src/main/java/okhttp3/internal/io/FileSystem.kt

```diff
@@ -50,24 +50,24 @@ interface FileSystem {
 
       @Throws(FileNotFoundException::class)
       override fun sink(file: File): Sink {
-        try {
-          return file.sink()
-        } catch (_: FileNotFoundException) {
-          // Maybe the parent directory doesn't exist? Try creating it first.
-          file.parentFile.mkdirs()
-          return file.sink()
-        }
+          return try {
+              file.sink()
+          } catch (_: FileNotFoundException) {
+              // Maybe the parent directory doesn't exist? Try creating it first.
+              file.parentFile.mkdirs()
+              file.sink()
```

Comment:
the only case I don’t really like is this one, which doesn’t jump out at me as being used as the return value